python_scripts/02_numerical_pipeline_introduction.py

```diff
@@ -11,20 +11,23 @@
 # In this notebook, we present how to build predictive models on tabular
 # datasets, with only numerical features.
 #
-# In particular we will highlight:
+# In particular we highlight:
 #
 # * the scikit-learn API: `.fit(X, y)`/`.predict(X)`/`.score(X, y)`;
 # * how to evaluate the generalization performance of a model with a train-test
 #   split.
 #
+# Here API stands for "Application Programming Interface" and refers to a set of
+# conventions to build self-consistent software. Notice that you can visit the
+# Glossary for more info on technical jargon.
```

Comment:
Shouldn't we link to the Glossary here?

Reply:
We can link to the [Glossary on the jupyter book](https://inria.github.io/scikit-learn-mooc/appendix/glossary.html), which will take FUN users out of the platform, or just encourage them by words to use the glossary inside FUN. What do you prefer?

Reply:
Indeed. I forgot there was no way to put a relative link that would work for both. Let's keep it unchanged then.